src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
+    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.BlazorWebAssembly" />
```

Comment:
When adding new manifests the only change necessary should be to include the package id, version, and workload id here. @jonathanpeppers  @pranavkm is the existing information correct (including the versions in verion.props)? How will we be keeping versions up to date?

Reply:
For Blazor we will want to set up dependency flow.  For Xamarin the updates will be manual, and only needed once per feature band (as installing the workload will install the updated manifest with the right version numbers).